client/my-sites/plugins/index.js

```diff
@@ -35,6 +36,16 @@ module.exports = function() {
 	}
 
 	if ( config.isEnabled( 'manage/plugins' ) ) {
+		page( '/plugins/wpcom-masterbar-redirect/:site', context => {
+			analytics.tracks.recordEvent( 'calypso_wpcom_masterbar_plugins_view_redirect' );
```

Comment:
Maybe this could be named moreso about the button click in the masterbar menu?

`calypso_wpcom_masterbar_plugins_add_click`